hack/update-codegen.sh

```diff
@@ -82,3 +85,5 @@ ${REPO_ROOT_DIR}/hack/update-deps.sh
 
 # Make sure the OpenAPI specification and Swagger file are up-to-date
 ${REPO_ROOT_DIR}/hack/update-openapigen.sh
+
+shim_gopath_clean
```

Comment:
Maybe `trap shim_gopath_clean EXIT` and move this above, so this gets done even if the script is interrupted? Otherwise lgtm, and thanks for doing this!

Reply:
That works well, cheers!